Fix main screen layout

solita-hjalonen opened the pull request:
* Fix exposures element icon size and shadow properties
* Fix incorrect spacings and issues with ambiguous scroll content height

Commit:
Fix exposures element icon size and shadow

Commit:
Fix spacing and ambiguous scroll content issues

Commit:
Fix status image size and animation from clipping

Commit:
Fix scrolling issues

- Add always bouncing behaviour to scroll views to prevent the large title from being hidden.
- Remove old scrollview
- Fix additional layout issues

Commit:
Update CHANGELOG.md

TomiLahtinen commented on Koronavilkku/Koronavilkku/View/Home/StatusHeaderView.swift:
```diff
@@ -128,14 +130,18 @@ final class StatusHeaderView: UIView {
     }
     
-    func adjustSize(by: CGFloat) {
+    func adjustSize(by: CGFloat, topInset: CGFloat) {
```
Maybe add comments on how this works and why it does what it does and/or UI tests

TomiLahtinen commented on Koronavilkku/Koronavilkku/View/Home/StatusHeaderView.swift:
```diff
@@ -160,8 +166,8 @@ final class StatusHeaderView: UIView {
-            make.height.centerX.equalToSuperview()
-            make.width.equalTo(radarView.snp.height)
+            make.top.height.centerX.equalToSuperview()
+            make.width.equalTo(radarView.snp.height).multipliedBy(118.0 / 138.0)
```
Move these magic numbers to constants please

Commit:
Review corrections

- Use a constant for the magic number
- Add comments to explain what adjustSize() does

Commit:
Fix UI elevation effects

- Removed old, unused elevation helper
- Fix CardElement & RoundedButton shadows
- Add shadowPath optimisations

Commit:
Refactor and unify shadow effects

- Add new Elevation enum and UIView extension for setting shadow effects
- Move publish token code input logic into a subclass

Commit:
Update CHANGELOG.md

Commit:
Add comment

Commit:
Fix clipping issue in the symptoms element
